tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.php

```diff
@@ -178,44 +178,44 @@ public function testSwitchDefault($testMarker, $openerOffset, $closerOffset, $co
         $this->assertSame(
             $expectedScopeOpener,
             $tokenArray['scope_opener'],
-            sprintf('Scope opener of the T_DEFAULT token incorrect. Marker: %s.', $testMarker)
+            sprintf('Scope opener of the T_DEFAULT token incorrect. Marker: %s.', $openerMarker)
         );
         $this->assertSame(
             $expectedScopeCloser,
             $tokenArray['scope_closer'],
-            sprintf('Scope closer of the T_DEFAULT token incorrect. Marker: %s.', $testMarker)
+            sprintf('Scope closer of the T_DEFAULT token incorrect. Marker: %s.', $closerMarker)
```

Comment:
Shouldn't these messages still refer to `$testMarker` ? as they are about the keys set on the `T_DEFAULT` token (which is found via the `$testMarker`).

Reply:
Good point, I missed that. It is now fixed.